bcda/api.go

```diff
@@ -373,9 +356,9 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 */
 func serveData(w http.ResponseWriter, r *http.Request) {
 	dataDir := os.Getenv("FHIR_PAYLOAD_DIR")
-	acoID := chi.URLParam(r, "acoID")
-	jobID := chi.URLParam(r, "jobID")
-	http.ServeFile(w, r, fmt.Sprintf("%s/%s/%s.ndjson", dataDir, jobID, acoID))
+	fileName := chi.URLParam(r, "fileName")
+	jobID := chi.URLParam(r, "jobId")
```

Comment:
Same as comment in router.go-- let's keep `jobID` to be consistent.

Reply:
Same comment as above.